org.darkdimension.rue.json

```diff
@@ -0,0 +1,53 @@
+{
+	"id": "org.darkdimension.rue",
+	"runtime": "org.freedesktop.Platform",
+	"runtime-version": "20.08",
+	"sdk": "org.freedesktop.Sdk",
+	"command": "rue",
+	"modules": [
+		"shared-modules/glew/glew.json",
```

Comment:
you need to add the `shared-modules` git submodule for this to work.

Reply:
Thanks, I've added `shared-modules` as a submodule as part of this pull request.